py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -81,6 +81,23 @@ def check_response(self, response):
         """
         Checks that a JSON response from the WebDriver does not have an error.
 
+        If an element is no longer part of the DOM a StaleElementReferenceException
```

Comment:
not really a good reason to add this py doc here. Most users shouldn't be calling (or even looking at) the ErrorHandler class, it's mostly an internal class to handle responses from the server